api/templates/index.html

```diff
@@ -2,63 +2,25 @@
 {% set active_page = "index" %}
 
 {% block title %}URL Shortener {% endblock %}
-{% block head %}
-{{ super() }}
-{% endblock %}
-
-
-
 
 <div class="container">
-{% block header %}{% endblock %}
+  {% block header %}{% endblock %}
 </div>
 
 {% block page_content %}
-<div class="container">
-<h1>Welcome to the URL Shortener</h1>
-<br><br><br>
     <form action="/" method="post">
-  <div class="form-group">
-    <label for="OriginalUrl">{{ data.label }}</label>
+     <div class="form-group">
+      <label for="OriginalUrl">{{ data.label }}</label>
+      <br>
+      <h1>GC URL Shortener</h1>
+      <input class="form-control form-control-lg" type="text" id="original_url" name="original_url"  value="{{ data.url }}" placeholder="Shorten your link">
```

Comment:
Can this type be changed to url
This allows certain browsers to optimize the text input.
For example it will not capitalize to Http:// and will being up a keyboard more suitable for url entry on mobile devices.

I quickly checked on iOS simulator
<img width="561" alt="Screen Shot 2022-03-25 at 10 32 42 AM" src="https://user-images.githubusercontent.com/8114/160130854-be55ba0a-18b3-4ea2-8189-dd29379c2661.png">

Compared to
<img width="561" alt="Screen Shot 2022-03-25 at 10 35 13 AM" src="https://user-images.githubusercontent.com/8114/160130924-814ade07-08a7-4feb-9b80-bc07872f88e8.png">

Reply:
Good point! I have changed this to type of "url"